pipelines/jobs/pipeline_job_template.groovy

```diff
@@ -58,6 +58,7 @@ pipelineJob("${BUILD_FOLDER}/${JOB_NAME}") {
 
     parameters {
         textParam('targetConfigurations', JsonOutput.prettyPrint(JsonOutput.toJson(targetConfigurations)))
+        stringParam('activeNodeTimeout', "0", 'Number of minutes we will wait for a label-matching node to become active.')
```

Comment:
Are we happy having the default timeout value as 0? This would mean that the nightlies will fail automatically if a node isn't found straight away (and with most of them executing at the same time, it's very possible some platforms which we don't have a lot of machines for will crash out instantly)

Reply:
A month ago test pipelines were changed (see https://github.com/AdoptOpenJDK/openjdk-tests/pull/1950) to abort immediately (no timeout) and to date we have seen no issues.  Given this, I think default of 0 is fine.  Perhaps we should have done this in stages, where we first just exit with no activeNodeTimeout (and see if it was actually even needed).

Note, the check is if nothing matches the LABEL being searched for, not if a machine is busy.